### PR TITLE
[DPR2-132] Use deterministic image tag

### PR DESCRIPTION
**Pull request by stalwartsolutions**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -46,10 +56,15 @@ workflows:
           name: helm_lint
       - hmpps/build_multiplatform_docker:
           name: build_docker
+          additional_docker_build_args: --tag "quay.io/hmpps/${CIRCLE_PROJECT_REPONAME}:Build.${CIRCLE_WORKFLOW_ID}"
```
Comment (gavriil-g-moj) [inline .circleci/config.yml:59]: Just to note that in the reporting mi application this has the pattern of: "quay.io/hmpps/${CIRCLE_PROJECT_REPONAME}".

Comment (gavriil-g-moj) [reply .circleci/config.yml:59]: Would we need to keep the same or does it not matter?

Comment (stalwartsolutions) [reply .circleci/config.yml:59]: For the image name, yes, but this is also the app version. Like this: https://github.com/ministryofjustice/hmpps-digital-prison-reporting-mi/pull/107/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R92

Comment (gavriil-g-moj) [reply .circleci/config.yml:59]: Ah thanks that makes sense.

**Review by gavriil-g-moj — APPROVED** [review]
LGTM

**Pull request merged**